src/blueprints/tv2/timeline-object-factories/tv2-viz-timeline-object-factory.ts

```diff
@@ -11,31 +12,31 @@ import {
 } from '../../timeline-state-resolver-types/viz-types'
 import { Tv2GraphicsLayer } from '../value-objects/tv2-layers'
 import { DeviceType } from '../../../model/enums/device-type'
-import { TimelineObject } from '../../../model/entities/timeline-object'
 import {
   Tv2FullscreenGraphicsManifestData,
   Tv2OverlayGraphicsManifestData
 } from '../value-objects/tv2-action-manifest-data'
+import { Tv2GraphicsCommandTimelineObjectFactory } from './interfaces/tv2-graphics-command-timeline-object-factory'
 
-
-enum ChannelName {
+enum Engine {
```

Comment:
Engine seems to be referring to a broader data structure than a name/identifier. I suggest to rename this to `EngineName`.

Reply:
Updated.